plugins/catalog-backend-module-ldap/src/ldap/config.ts

```diff
@@ -161,6 +165,26 @@ export type GroupConfig = {
   };
 };
 
+/**
+ * Configuration for LDAP vendor-specific attributes.
+ *
+ * Allows custom attribute names for distinguished names (DN) and
+ * universally unique identifiers (UUID) in LDAP directories.
+ *
+ * @public
+ */
+export type VendorConfig = {
+  /**
+   * Attribute name for the distinguished name (DN) of an entry,
+   */
+  dnAttributeName: string;
+
+  /**
+   * Attribute name for the unique identifier (UUID) of an entry,
+   */
+  uuidAttributeName: string;
```

Comment:
Just to double-check are both configurations required?

Reply:
I don't think so they should be required, updated the code to make them optional.